### DISCUSSION TITLE
Understanding training spacy models

### DISCUSSION THREAD

**Discussion by JessitaMS** [explosion/spaCy#10711] [Help: Model Advice] [answered]
Now I have another doubt @adrianeboyd  I understand that to train the model, the more text it has, the better it will be able to learn the labels, but for example with the model es_core_news_lg, I have seen that PERSON labels many entities that actually corresponds to this label, but I understand that it may not have in its data all the names that I work with in my dataset.

So my question is, why with the data and labels that my model trained, is it not able to learn to associate another name with which it has not been trained, I do not know that it should be able to learn, without specifically putting all the cases? .

If you can help me understand this a bit, since I'm very new to the subject, thank you!

**Comment by polm** [comment, marked as answer]
See #10724, which seems to be a duplicate of this.
